inst/emails/contact_aon_recurring

```diff
@@ -1,12 +1,12 @@
 Dear %s,
 
-The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
+The NSF-funded Arctic Data Center support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
 
-If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
+Submissions to the Arctic Data Centre can be done here: https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
```

Comment:
"here' links are hard to follow. Better to use a more descriptive phrase, like "can be done via our online web form: https..."

Reply:
The `https://arcticdata.io/catalog/#share` URL still works, but is old. We should use `https://arcticdata.io/catalog/submit`

Reply:
Also, fix `Centre` typo 🙂

Reply:
> "here' links are hard to follow. Better to use a more descriptive phrase, like "can be done via our online web form: https..."

Applies throughout - e.g. to the portals page link